core/src/cats/sql_update.cc

```diff
@@ -563,6 +563,14 @@ void BareosDb::UpgradeCopies(const char* jobids)
 
   SqlQuery(query.c_str());
 
+  // remove the worker job that copied it
+  Mmsg(query,
+       "DELETE FROM Job "
+       "WHERE Type='%c' "
+       "AND priorjobid IN ( SELECT JobId FROM cpy_tmp )",
+       JT_COPY);
+  SqlQuery(query.c_str());
+
   Mmsg(query, "DROP TABLE cpy_tmp");
   SqlQuery(query.c_str());
```

Comment:
```suggestion
  // remove the worker job that copied it
  SqlQuery(
       "DELETE FROM Job "
       "WHERE Type='JT_COPY' "
       "AND priorjobid IN ( SELECT JobId FROM cpy_tmp )");
```

Reply:
most of it is fixed, but `JT_COPY` is not. If for any reason in the future we decide to change what `JT_COPY` refers to, the result would be wrong

Reply:
Agreed